Emby.IsoMounting/IsoMounter/LinuxIsoManager.cs

```diff
@@ -144,24 +144,21 @@ public Task Install(CancellationToken cancellationToken)
             return Task.FromResult(false);
         }
 
-        public async Task<IIsoMount> Mount(string isoPath, CancellationToken cancellationToken)
+        public Task<IIsoMount> Mount(string isoPath, CancellationToken cancellationToken)
```

Comment:
Could this just return IIsoMount if nothing is being done async?

Reply:
That would mean changing the Interface and everything that implements that.